lib/Newsletter/ViewInBrowser/ViewInBrowserRenderer.php

```diff
@@ -98,20 +95,16 @@ public function render(
 
   /** this is here to prepare entities for the shortcodes library, when this whole file uses doctrine, this can be deleted */
```

Comment:
This comment says that the function can be deleted but don't we have to still assign the data to the shortcodes?

Reply:
I agree. In my opinion, we cannot delete it. But maybe I overlooked something.

Reply:
Thanks for confirming, I removed the comment.